src/components/classic/documentation/sitemap.md

```diff
@@ -470,7 +470,7 @@ Developers
 *   [Sandbox](sandbox)
 *   [Source](source)
 *   [Source XRef](source-xref)
-*   [SVN](svn)
+*   [SVN](git)
```

Comment:
SVN -> Git

Reply:
Updated the link without the title. Thanks for catching it